### ISSUE TITLE
Notifications for Task lists

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/tasks#638]
Wenn a Task is on due
when a new Task item is added to a shared list

**Comment by hundsboog** [comment]
Killer feature!

**Comment by Torenga** [comment]
And when a task starts (start date) please.

**Comment by caemor** [comment]
Is there any work done for this already?
If not, how complicated would it be to make a pr for this?

This seems to be the workflow needed to be done:
https://github.com/nextcloud/notifications/blob/master/docs/notification-workflow.md

**Comment by macpac59** [comment]
Oh, the feature was postponed again. :(
If only this would work (no matter whether it's a browser or desktop notification), I'm pretty sure the productivity of my collective would rise. If there's a way to push this topic, give me a hint, @raimund-schluessler.

**Comment by raimund-schluessler** [comment]
Yes, I would agree that a notification should only be triggered by a reminder.
